src/server.c

```diff
@@ -4601,6 +4605,28 @@ sds genRedisInfoString(const char *section) {
                                   0, /* not a crash report */
                                   sections);
     }
+
+    /* Threads */
+    if (allsections || defsections || !strcasecmp(section,"threads")) {
+        if (sections++) info = sdscat(info,"\r\n");
+        info = sdscatprintf(info,
+        "# Threads\r\n"
+        "io_threads:%d\r\n"
+        "thread_reads_enabled:%d\r\n"
+        "io_threads_active:%d\r\n"
+        "io_threaded_reads_processed:%lld\r\n"
+        "io_threaded_writes_processed:%lld\r\n"
+        "single_threaded_reads_processed:%lld\r\n"
+        "single_threaded_writes_processed:%lld\r\n",
+        server.io_threads_num,
+        server.io_threads_do_reads,
+        server.io_threads_active,
+        server.stat_io_reads_processed,
+        server.stat_io_writes_processed,
+        server.stat_total_reads_processed - server.stat_io_reads_processed,
+        server.stat_total_writes_processed - server.stat_io_writes_processed);
```

Comment:
maybe these 4 metrics should be moved to the "stats" section?
and so really want to show "single" vs "io_threaded", or just show "single" vs "total" (or "io_threaded" vs "total")?
I think showing "total" and "single" makes the most sense.
@madolson what do you think?

Reply:
another related thought, we don't usually relay the configuration parameters in info (i.e. `io_threads_num` and `io_threads_do_reads`).
there are some exceptions (e.g. `maxmemory`) , but i'm not sure if this one is needed.
so i think it would be better to move the `io_threads_active` to the "server" section, move the stats to the "stats" section, and drop the rest and the new section.

Reply:
For stats, I'm fine moving them. I sort of think the # of io_threaded ops should be in this section, but the total ops don't really make sense. Since they should probably stick together, I think it makes sense to move them both to stats.  

If we want to show something + total, I think total + io_threaded makes the most sense. The value here is understanding if you are effectively using the threading or not, so it will be 0 for most people.

The original intention for this PR was for it to be propagated out to the crash log to rule out race conditions. I thought that was sensible idea and just suggested some additional added value. There are also some things like cluster mode that are shown, so I think it makes sense to keep that.

Reply:
Wouldn't we get all information we need from just showing "single" vs "io_threaded" ? ie if the user wants to know the total of write events, he/she would just add single_thread_writes and io_threaded_writes.

Regarding configuration parameters: is there another way a user can get those information from? It would help with identifying why they are getting certain values in `io_threaded` vs `single` if we relay the configuration information to the user.

Reply:
yes, of course the user can get the total by summing, i'm just saying that for stats, i think it is better to show the total too. personally i think showing `total` and `single` is right (maybe named `main_thread_reads_processed` and `total_reads_processed`), but if you think you want to show `io_threaded` so that it's easy to realize it's 0 right away, i'm ok with whatever you guys choose (can be `total` and `io_threaded`).

the user can get the config by doing `CONFIG GET *`, we **don't** normally want to expose the configuration in the info. there some some exceptions where that configuration is needed to complement other info fields (e.g. used_mem vs maxmemory), or maybe cases that we need it present in crash log (although can easily add the entire configuration to the crash log too).

it seems to me that we want to drop the `# Threads` info section entirely. stats belong in the `# Stats` section (where there are various stats of different subsystems), and i feel the `io_threads_active` needs to go into the `# Server` section.

Reply:
"(although can easily add the entire configuration to the crash log too)" <- I like this idea

With that, I vote that:
1. We add io_threads_active to #server
2. Add total_reads_processed/total_writes_processed/io_threaded_reads_processed,io_threaded_writes processed to #stats
3. Remove the configs, and move them to a separate section we'll define later.

Reply:
That makes sense! I will update the PR accordingly.